client/src/components/styles/FormInput.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components'
+
+const FormInput = styled.input`
+  width: 100%;
+  height: 40px;
+  padding: 6px 12px;
+  margin: 5px 0;
+  background-color: #fff;
+  background-image: none;
+  border: 1px solid #d9dadc;
```

Comment:
Move all colors into one place so it is easy to to theme update later on if needed

Reply:
Done!